libraries/SITL/SIM_GPS_Trimble.cpp

```diff
@@ -204,13 +204,13 @@ void GPS_Trimble::publish(const GPS_Data *d)
             } dop {};
             // Check the payload size calculation in the compiler
             constexpr auto dop_size = sizeof(gsof_dop);
-            static_assert(dop_size == 18);
+            static_assert(dop_size == 18, "gsof_dop must be 8 bytes");
             constexpr auto dop_record_type_size = sizeof(gsof_dop::OUTPUT_RECORD_TYPE);
-            static_assert(dop_record_type_size == 1);
+            static_assert(dop_record_type_size == 1, "OUTPUT_RECORD_TYPE must be 1 byte");
             constexpr auto len_size = sizeof(gsof_dop::RECORD_LEN);
-            static_assert(len_size == 1);
+            static_assert(len_size == 1, "gsofRECORD_LEN_dop must be 1 bytes");
```

Comment:
little white space issue here.  By the way, do we want to call it Trimble instead of GSOF?

Reply:
Think I have fixed